domaintools/api.py

```diff
@@ -376,8 +375,6 @@ def iris_detect_new_domains(self, monitor_id=None, tlds=None, risk_score_ranges=
                                 order=None, include_domain_data=False, offset=0, limit=None, preview=None, **kwargs):
         """Returns back a list of new domains in Iris Detect based on the provided filters.
         """
-        if discovered_since and changed_since:
-            raise ValueError('Please only use one of discovered_since or changed_since in your query.')
```

Comment:
I changed my mind on this because if the user really wants to do this we might as well let them.